packages/cozy-sharing/src/components/EditLinkPermissionDialog.jsx

```diff
@@ -62,12 +62,20 @@ const EditLinkPermissionDialog = ({
         >
           <FormControlLabel
             value="true"
-            label={t('Share.permissionLink.read')}
+            label={
+              document.type === 'file'
```

Comment:
Can't you have a document without `type`? `type` is only there for `io.cozy.files` but we can share others data.

Reply:
Right! And now file text is default version.